### PR TITLE
LayoutAnimation API

### PR DESCRIPTION
**Pull request by sgny**

### CODE REVIEW

File: reason-react-native/src/apis/LayoutAnimation.re [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    ~create: animationConfig=?,
+    ~update: animationConfig=?,
+    ~delete: animationConfig=?,
+    unit
+  ) =>
+  layoutAnimationConfig =
+  "";
+
+[@bs.module "react-native"] [@bs.scope "LayoutAnimation"]
+external configureNext:
```
Comment (cknitt) [inline reason-react-native/src/apis/LayoutAnimation.re]: Maybe we should have two bindings `configureNext` and `configureNextWithEndCallback`?

Comment (sgny) [reply reason-react-native/src/apis/LayoutAnimation.re]: Further to your comment elsewhere regarding searching for such sister functions, perhaps we should simply add comments indicating that we have a convenience function whenever we do something like this?

**Comment by sgny** [comment]
I added `Presets` so that presets like `linear` or `spring` may be passed as argument to `configureNext` and `configureNextWithCallback`. Basically,
```
LayoutAnimation.configureNext(LayoutAnimation.Presets.linear)
```
is the same as
```
LayoutAnimation.linear()
```
but they may nevertheless prove useful when making use of a callback.

Presets is actually a dictionary, so we could bind to `Presets` instead and define getters with `[@bs.get]` but this PR allows BS API to be mostly identical to the JS API.

File: reason-react-native/src/apis/LayoutAnimation.re (hunk 1 above)
Comment (cknitt) [reply reason-react-native/src/apis/LayoutAnimation.re]: Yes, why not, I'd be happy to approve a (separate) PR adding such comments to all the xxxWithYYY functions. 👍

**Pull request merged**